runtime/sema/type.go

```diff
@@ -4854,15 +4862,14 @@ func (t *AddressType) GetMembers() map[string]MemberResolver {
 // However, to check if a type *strictly* belongs to a certain category, then consider
 // using `IsSameTypeKind` method. e.g: "Is type `T` an Integer type?". Using this method
 // for the later use-case may produce incorrect results.
-//   * IsSubType()      - To check the assignability. e.g: Is argument type T is a sub-type
-//                        of parameter type R. This is the more frequent use-case.
-//   * IsSameTypeKind() - To check if a type strictly belongs to a certain category. e.g: Is the
-//                        expression type T is any of the integer types, but nothing else.
-//                        Another way to check is, asking the question of "if the subType is Never,
-//                        should the check still pass?". A common code-smell for potential incorrect
-//                        usage is, using IsSubType() method with a constant/pre-defined superType.
-//                        e.g: IsSubType(<<someType>>, FixedPointType)
-//
+//   - IsSubType()      - To check the assignability. e.g: Is argument type T is a sub-type
+//     of parameter type R. This is the more frequent use-case.
+//   - IsSameTypeKind() - To check if a type strictly belongs to a certain category. e.g: Is the
+//     expression type T is any of the integer types, but nothing else.
+//     Another way to check is, asking the question of "if the subType is Never,
+//     should the check still pass?". A common code-smell for potential incorrect
+//     usage is, using IsSubType() method with a constant/pre-defined superType.
+//     e.g: IsSubType(<<someType>>, FixedPointType)
```

Comment:
Why did this get reformatted?

Reply:
unscrupulous gofmt on save

Reply:
Since Go 1.19, gofmt also re-formats doc comments: https://tip.golang.org/doc/go1.19#go-doc.

Please revert the changes for now and use Go 1.18 for formatting, and then let's have a separate PR where we migrate to Go 1.19 and re-format the doc strings manually. This automated re-format makes the comments unreadable.